main.py

```diff
@@ -1,9 +1,12 @@
 import logging
 
-from living_documentation_generator.action.action_inputs import ActionInputs
+from github import Github, Auth
+
+from living_documentation_generator.action_inputs import ActionInputs
 from living_documentation_generator.generator import LivingDocumentationGenerator
-from living_documentation_generator.github_integration.github_manager import GithubManager
-from living_documentation_generator.utils import set_action_output
+
+from living_documentation_generator.github_projects import GithubProjects
+from living_documentation_generator.utils.utils import set_action_output
```

Comment:
I would reconsider deviding these four imports. For me `GithubProjects` should be on the same level as `ActionInputs`, they are doing similar operartions.

Reply:
I do not get it.

Reply:
To delete empty line number 7, because all are imports from other classes that we created in our project. But I might be wrong. I just don't see the separation to have these four imports in two groups.